src/datasource/gateway.test.js

```diff
@@ -0,0 +1,310 @@
+import { readFileSync } from 'fs';
+import {
+  getGatewayClient
+} from "./gateway";
+import { 
+  processTx
+} from "./gateway-test-utils";
+
+const testConfig = {
+  "gatewayUrl": "http://localhost:3000/v1/60676c9f7cbbfe002f0b9cbe",
+  "http": {
+    "timeout": 0,
+    "headers": {
+      "Content-Type": "application/json",
+      "Blockchain-Subdomain": "mainnet",
+    }
+  },
+  "ppkFilePath": `${__dirname}/ppk.json`,
+  "passphrase": "L0c4ld3v3l0pment!",
+  "chainId": "mainnet",
+  "txFee": 10000,
+}
```

Comment:
This was hardcoded in here for convenience's sake. I will add a bit of `||` to allow for using `.env.test` if you anyone fancies that. Will prove useful in CI to use env variables in instead.